### PR TITLE
fix: use explicit FOURC_SILENCE_LOGGING for silencing logging

### PR DESCRIPTION
**Pull request by c0state**
I think being explicit makes more sense, no?

Tries to address #5 

Technically breaking and not sure what impact it has on current usages though...

### CODE REVIEW

**Comment by jquense** [comment]
I'm not sure about this, the reason it's checking for TEST is so unrelated tests are bogged down with logging, It's a nice default for ppl downstream that don't necessarily know how to toggle this off.

I think the concern is just that we don't want test suites full of debug messages. Maybe worth adding a variable but defaulting it to false in TEST, that way it's easy to override?

File: createLogging.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ module.exports = function createLogging({
   Transport = transports.Console,
 } = {}) {
   const container = new Container();
-  const TEST = Env.get('NODE_ENV', '') === 'test';
+  const silenceLogger = Env.get.boolish('FOURC_SILENCE_LOGGING', true);
```
Comment (jquense) [inline createLogging.js]:
```suggestion
  const silenceLogger = Env.get.boolish('FOURC_SILENCE_LOGGING', Env.get('NODE_ENV', '') === 'test');
```

was thinking more this

Comment (c0state) [reply createLogging.js]: Hmm it seems OK to decouple the log silencing logic from NODE_ENV and default to true to suppress (as is the current behavior). Only downside I think is for people that actually set `NODE_ENV=test` to suppress logging which would be the odd case I think.

Comment (c0state) [reply createLogging.js]: Actually, never mind, the default was to *not* suppress unless NODE_ENV==='test'. I thought the original suppressed by default. So this looks good I think--it should preserve existing behavior and allow for more explicit suppression via the new `FOURC_SILENCE_LOGGING` env var.

**Comment by c0state** [comment]
Also, I don't think the tests work correctly. `beforeEach` is setting NODE_ENV to `test` (thus suppressing) but the tests all check that logs are being written unless I'm mistaken.

```
  beforeEach(() => {
	...
    process.env.NODE_ENV = 'test';
  });
```

**Comment by jquense** [comment]
The tests are checking if the logging functions are called, the output is being supossed so that it's not _also_ written to console.

**Review by jquense — APPROVED** [review]

**Comment by c0state** [comment]
> The tests are checking if the logging functions are called, the output is being supossed so that it's not _also_ written to console.

Ah right

**Pull request merged**